src/Cool/DidYouKnow/didYouKnowModal.hpp

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+#include <vector>
+#include "cereal/types/chrono.hpp"
+
+static constexpr auto all_tips = std::array{
```

Comment:
Lab should be the one creating this array as we might want to add tips that are specific to Lab. To pass it to this class you can use `std::span`.